### PR TITLE
fix: apply theme's token to LoadingProvider component

### PR DESCRIPTION
**Pull request by minagishl**
Github issue #1100 

## Description

apply theme's token to LoadingProvider component

## Current behavior (updates)

no change

## New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

**Review by taroj1205 — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Please include the `changeset` for the `patch`!

**Comment by minagishl** [comment]
@taroj1205 
I ran `pnpm changeset`, please confirm.

File: packages/components/loading/src/loading-provider.tsx [hunk 1]
```diff
@@ -358,14 +358,14 @@ const getOverlayStyle = (
   right: 0,
   bottom: 0,
   left: 0,
-  zIndex: 9999,
+  zIndex: "fallback(full, 9999)",
```
Comment (taroj1205) [inline packages/components/loading/src/loading-provider.tsx:361]: `full` is not a token for zIndex. Please use `beerus` instead.

File: packages/components/loading/src/loading-provider.tsx [hunk 2]
```diff
@@ -478,20 +478,20 @@ const BackgroundComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       position: "fixed",
-      right: "1rem",
-      bottom: "1rem",
-      zIndex: 9999,
+      right: "fallback(4, 1rem)",
+      bottom: "fallback(4, 1rem)",
+      zIndex: "fallback(full, 9999)",
```
Comment (taroj1205) [inline packages/components/loading/src/loading-provider.tsx:483]: `full` is not a token for `zIndex`. Please use `beerus`.

**Review by taroj1205 — COMMENTED** [review]
Thanks! There are a few changes needed. Please fix it.

**Comment by minagishl** [comment]
@taroj1205 
Corrected, please confirm.

**Review by taroj1205 — COMMENTED** [review]
LGTM

**Review by taroj1205 — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**